contracts/test-helpers/Helper_GasMeasurer.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.7.0;
+
+contract Helper_GasMeasurer {
+    function measureCallGas(
+        address _target,
+        bytes memory _data
+    )
+        public
+        returns ( uint )
+    {
+        uint gasBefore;
+        uint gasAfter;
+
+        uint calldataStart;
+        uint calldataLength;
+        assembly {
+            calldataStart := add(_data,0x20)
+            calldataLength := mload(_data)
+        }
+
+        bool success;
+        assembly {
+            gasBefore := gas()
+            success := call(gas(), _target, 0, calldataStart, calldataLength, 0, 0)
+            gasAfter := gas()
```

Comment:
I'm not sure exactly how this overhead works, but presumably will need to subtract at minimum the cost of `GAS + `CALL`, and possibly more, to get this to be the value we consume in geth.